Gemfile

```diff
@@ -1,7 +1,7 @@
 source "http://rubygems.org"
 
 gem 'multi_json'
-gem 'omniauth-oauth', '~> 1.0'
+gem 'omniauth', '~> 1.3', '>= 1.3.1'
```

Comment:
The `omniauth` gem is more updated and have continuous fix for issues recently than the `omniauth-oauth` last update was in 2015